ppx/reason_react_ppx.ml

```diff
@@ -1093,8 +1097,8 @@ let jsxMapper =
             let namedTypeList = List.fold_left argToType [] namedArgList in
             let loc = gloc in
             let externalDecl =
-              makeExternalDecl fnName loc namedArgListWithKeyAndRef
-                namedTypeList
+              makeExternalDecl ~component_is_external:false fnName loc
```

Comment:
We might want to add `component_is_external` to `makeExternalDecl` declaration params to avoid confusing errors with currying down the line.